Dockerfile

```diff
@@ -16,21 +16,23 @@ ENV LOUNGE_SRC "${LOUNGE_HOME}/src"
 RUN groupadd --gid ${gid} ${group} \
       && useradd --home "${LOUNGE_HOME}" --create-home --uid ${uid} --gid ${gid} ${user}
 
-VOLUME "${LOUNGE_DATA}"
-RUN chown -R ${user} "${LOUNGE_DATA}"
-
-# Drop root.
-USER ${user}
-
 # Install thelounge.
 RUN mkdir -p "${LOUNGE_SRC}"
 WORKDIR "${LOUNGE_SRC}"
 RUN curl -L $(npm view thelounge@${LOUNGE_VERSION} dist.tarball) | tar --strip-components 1 -xzf -
 RUN npm install
+RUN npm link
```

Comment:
I prefer doing it this way with Docker for the same reasons [as before](https://github.com/thelounge/docker-lounge/pull/7#discussion_r62414098). But I understand what you're getting at (and the benefits of just doing a global npm installation, although I don't agree with it being an issue right now :smile:). I'll have to fix some stuff in the `Dockerfile` for the `2.0.0` release, I'll take that opportunity and move stuff to a global installation then as well.